File: Programacionavanzada/ejercicios/2243330341_ActClase27Feb_Practica2/src/main/PracticaFinal.java
package main;

import controller.AppService;
import controller.MainController;
import model.StorageType;
import view.MenuPrincipalFrame;

import javax.swing.SwingUtilities;

public class PracticaFinal {
    public static void main(String[] args) {
        SwingUtilities.invokeLater(() -> {
            AppService service = new AppService();
            service.setStorageType(StorageType.TXT);
            MenuPrincipalFrame frame = new MenuPrincipalFrame();
            new MainController(frame, service);
            frame.setVisible(true);
        });
    }
}

File: Programacionavanzada/ejercicios/2243330341_ActClase27Feb_Practica2/src/2243330341_Practica2_python/main/practica_final.py
from PA_a2251330007_practica2_python.controller.app_service import AppService
from PA_a2251330007_practica2_python.controller.main_controller import MainController
from PA_a2251330007_practica2_python.model.storage_type import StorageType
from PA_a2251330007_practica2_python.view.menu_principal_frame import MenuPrincipalFrame


def main() -> None:
    service = AppService()
    service.set_storage_type(StorageType.TXT)
    frame = MenuPrincipalFrame()
    MainController(frame, service)
    frame.mainloop()


if __name__ == "__main__":
    main()
